src/components/data-management/SamplesTable.jsx

```diff
@@ -237,9 +237,9 @@ const SamplesTable = forwardRef((props, ref) => {
       // this situation should be included into the SamplesTable.jsx tests
       const sampleFiles = samples[sampleUuid]?.files || {};
 
-      const barcodesFile = sampleFiles['barcodes.tsv.gz'] ?? { upload: { status: UploadStatus.UPLOADING } };
-      const genesFile = sampleFiles['features.tsv.gz'] ?? { upload: { status: UploadStatus.UPLOADING } };
-      const matrixFile = sampleFiles['matrix.mtx.gz'] ?? { upload: { status: UploadStatus.UPLOADING } };
+      const barcodesFile = sampleFiles['barcodes.tsv.gz'] ?? { upload: { status: UploadStatus.FILE_NOT_FOUND } };
+      const genesFile = sampleFiles['features.tsv.gz'] ?? { upload: { status: UploadStatus.FILE_NOT_FOUND } };
+      const matrixFile = sampleFiles['matrix.mtx.gz'] ?? { upload: { status: UploadStatus.FILE_NOT_FOUND } };
```

Comment:
If you change this to `FILE_NOT_FOUND`, it will allow users to upload new data. Reading the comment above, I don't think this is the intended action. It seems we do not want the users to be able to upload new file. I'd ask @cosa65 about this,

Reply:
Why is it not intended? We had a ticket that allowed users to reupload missing files.

Reply:
I asked Martin he said he already changed it back to FILE_NOT_FOUND but has not merged it yet

Reply:
ooh I see. ok